testsuite/classlibrary/TestUnitTest.sc

```diff
@@ -44,7 +44,24 @@ TestUnitTest : UnitTest {
 		this.assertEquals( TestMixedBundleTester.findTestedClass, MixedBundleTester)
 	}
 
+	test_assertThrow {
+		this.assertThrow({ 1789.monarchy }, "assertThrow should return true for any error")
+	}
+
+	test_assertThrow_for_throw {
+		this.assertThrow({ \stone.throw }, "assertThrow should return true for thrown object")
+	}
 
+	test_assertThrow_with_specific_error {
```

Comment:
can you please follow the naming convention I suggested in my guide? That is,

```
test_assertThrow_withSpecificError
```

Most of the messages could also be eliminated, since they are implied from the name of the test.